Vagrantfile

```diff
@@ -153,14 +156,20 @@ Vagrant.configure("2") do |config|
       config.ignition.ip = ip
 
       # Uncomment below to enable NFS for sharing the host machine into the coreos-vagrant VM.
-      #config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+      #config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true, :mount_options => [ $nfs_options ]
+      $fstab_guts="### START #{$nfs_entrytag}\n"
       $shared_folders.each_with_index do |(host_folder, guest_folder), index|
-        config.vm.synced_folder host_folder.to_s, guest_folder.to_s, id: "core-share%02d" % index, nfs: true, mount_options: ['nolock,vers=3,udp']
+        config.vm.synced_folder host_folder.to_s, guest_folder.to_s, id: "core-share%02d" % index, nfs: true, mount_options: [ $nfs_options ]
+        $fstab_guts += "#{$nfs_server}:#{host_folder.to_s} #{guest_folder.to_s} nfs #{$nfs_options} 0 0\n"
       end
 
       if $share_home
-        config.vm.synced_folder ENV['HOME'], ENV['HOME'], id: "home", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+        config.vm.synced_folder ENV['HOME'], ENV['HOME'], id: "home", :nfs => true, :mount_options => [ $nfs_options ]
+        $fstab_guts += "#{$nfs_server}:#{ENV['HOME']} #{ENV['HOME']} nfs #{$nfs_options} 0 0\n"
       end
+      $fstab_guts += "### END #{$nfs_entrytag}\n"
+      # Create an /etc/fstab file for the nfs mounts.  Create the file if it doesn't exist.
+      config.vm.provision :shell, :inline => "touch /etc/fstab && sed -ei '/### START #{$nfs_entrytag}/,/### END #{$nfs_entrytag}/d' -e '$c#{$fstab_guts}'", :privileged => true
```

Comment:
Is the `/etc/fstab` inside the vagrant box only? Or, also mapped to host machine?

Reply:
It's in the vagrant box only.